packages/openapi-v3/README.md

```diff
@@ -43,24 +43,13 @@ then the `myControllerSpec` will be:
 
 ```js
 {
-    openapi: '3.0.0',
-    info: { title: 'LoopBack Application', version: '1.0.0' },
     paths: {
         '/greet': {
             get: {
-                responses:  {
-                    '200': {
-                        description: 'The string result.',
-                        schema: { type: 'string' }
-                    }
-                },
```

Comment:
Is it a valid controller spec when there are no responses described?

IMO, we should preserve the `200` response and describe it as returning `text/plain` content type.

Reply:
This is probably a bug, I manually wrote the code to generate the spec from `getControllerSpec()`. 

According to [the code in `controller-spec`](https://github.com/strongloop/loopback-next/blob/master/packages/openapi-v3/src/controller-spec.ts#L96)

the `responses` should be `{}`.

> we should preserve the 200 response and describe it as returning text/plain content type

Right now unless user provides `responses` spec in `@operation`/`@api`, we don't generate `responses` object for them....

Reply:
Related story: https://github.com/strongloop/loopback-next/issues/1086